src/main/java/stringCalculator/Calculator.java

```diff
@@ -0,0 +1,90 @@
+package stringCalculator;
+
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Calculator {
+
+    private String input;
+    private String[] parsedInput;
+    private Set<Character> validOperators;
+
+    Calculator(String input) {
+        this.input = input;
+
+        // Operator 검증을 위한 Set 설정
+        String operator = "+-*/";
+        validOperators = new HashSet<>();
+        for (int i = 0; i < operator.length(); i++)
+            validOperators.add(operator.charAt(i));
+
+        // input 검증
+        checkInputValidity();
+        parsedInput = input.split(" ");
+        checkParsedInputValidity();
+
+    }
+
+    private void checkInputValidity() {
+        if (input == null || input.isEmpty())
+            throw new IllegalArgumentException("Input is invalid form");
+    }
+
+    private void checkParsedInputValidity() {
+        checkOperatorValidity();
+        checkPair();
+    }
+
+    private void checkOperatorValidity() {
+        for (int idx = 1; idx < parsedInput.length; idx += 2)
+            checkOperator(parsedInput[idx]);
+    }
+
+    private void checkPair() {
+        if (parsedInput.length % 2 == 0)
+            throw new IllegalArgumentException("Input doesn't match pair");
+    }
+
+    private void checkOperator(String input) {
+        if (!(input.length() == 1 && validOperators.contains(input.charAt(0))))
+            throw new IllegalArgumentException();
+    }
+
+    int execute() {
+        int result;
+
+        result = calculate();
+
+        return result;
```

Comment:
```suggestion
        return calculate()
```
이거랑 다를게 없어보여요! 그렇다면 execute가 calulate가 되는건데 굳이 execute가 필요할까요?